src/navigation/NavigationRoot.tsx

```diff
@@ -20,22 +23,30 @@ const forFade = ({ current }: StackCardInterpolationProps) => ({
 
 const NavigationRoot = () => {
   const [showSplash, setShowSplash] = useState(true)
-  const isSignedIn = useSelector((state: RootState) => state.user.isSignedIn)
+  const { isBackedUp, isEducated, isRestored } = useSelector(
+    (state: RootState) => state.user,
+  )
+  const dispatch = useAppDispatch()
 
   useEffect(() => {
+    dispatch(restoreUser())
     setTimeout(() => {
       setShowSplash(false)
-    }, 750)
-  }, [])
+    }, 1250)
+  }, [dispatch])
```

Comment:
in what case does `dispatch` change value to trigger `useEffect` to execute again?

Reply:
It won't, but I'm just being "exhaustive" to appease eslint. I could just disable the eslint warning. It comes from the `eslint-plugin-react-hooks` library.